ambry-api/src/main/java/com.github.ambry/account/Container.java

```diff
@@ -445,6 +472,9 @@ JSONObject toJson() throws JSONException {
         metadata.put(MEDIA_SCAN_DISABLED_KEY, mediaScanDisabled);
         metadata.putOpt(REPLICATION_POLICY_KEY, replicationPolicy);
         metadata.put(TTL_REQUIRED_KEY, ttlRequired);
+        JSONArray contentTypeWhitelistForFilenamesOnDownloadJson = new JSONArray();
```

Comment:
How do you feel about using `putOpt` so that the json size does not increase for containers that don't have this set?

Reply:
`putOpt` may not work since empty sets may be used but this is a good suggestion. Will only add to json if non null and non empty